db/migrate/20200329222246_add_priority_column_to_notifications.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class AddPriorityColumnToNotifications < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+  def up
+    if !column_exists?(:notifications, :priority)
+      add_column :notifications, :priority, :integer, default: nil
```

Comment:
integer is huge... instead I would say since this is binary ... boolean is enough for now, maybe tinyint or byte.